src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx

```diff
@@ -601,41 +640,10 @@ describe('containers/pages/FocusInvestigation/activeMap', () => {
       </Provider>
     );
     await new Promise<unknown>(resolve => setImmediate(resolve));
-    expect(displayErrorMock.mock.calls.length).toBe(1);
+    expect(displayErrorMock).toHaveBeenCalled();
```

Comment:
Can we ascertain, in this test, how many times this was called?

Reply:
ok.